test/addPolynomial.test.js

```diff
@@ -0,0 +1,16 @@
+const canAdd = require('../lib/checks/canAddLikeTermPolynomialNodes');
```

Comment:
can you name `require`s and filenames after the full function?

e.g. `canAddLikeTermPolynomialNodes.test.js`

and `const canAddLikeTermPolynomialNodes = require('../lib/checks/canAddLikeTermPolynomialNodes');`